### DISCUSSION TITLE
All vercel sites are blocked

### DISCUSSION THREAD

**Discussion by RakeshPotnuru** [vercel/community#7307] [Help]
### Summary

Hi, I’m not able to access vercel sites including vercel.com, nextjs.org and all of my projects from my wifi network. I think my ISP blocked vercel. How to fix this?
![Vercel](https://github.com/vercel/community/assets/70439799/b90b1338-49c0-47aa-ab05-7e44c98f6cf1)

### Steps to Reproduce

Blocked from ISP.

**Comment by amyegan** [comment]
Thanks for reaching out, @RakeshPotnuru. Our support engineers put together a script to run a series of checks related to network issues.  That will give us more info about the problem. You can find the script and instructions at https://github.com/vercel-support/vercel-connect-debug

If you are on a shared network, such as wifi at a company or university, you may need to contact your network administrator for help unblocking Vercel sites. If you're on your own network, then you may want to reach out to your ISP directly as they may be quicker to respond to their own customers than to outside requests.

**Comment by RakeshPotnuru** [comment]
**Update on this**: I talked with my ISP. They have no idea what I'm saying. So this is an issue with network service, it is out of my hands.

**Comment by RakeshPotnuru** [comment]
**Update:** This issue is getting bigger. I submitted my website for verification to integrate the payment system with Razorpay and they are saying they can't access it.

[a comment by an automated account is not shown]
